bgpd/bgp_updgrp_packet.c

```diff
@@ -401,6 +402,8 @@ struct stream *bpacket_reformat_for_peer(struct bpacket *pkt,
 		afi_t nhafi;
 		int route_map_sets_nh;
 		nhlen = stream_getc_from(s, vec->offset);
+		filter = &peer->filter[paf->afi][SAFI_UNICAST];
```

Comment:
not `paf->safi` here as well?

Reply:
That makes sense.  I didn't consider other safi's would be setting the route-maps on peers but that is certainly possible.   I'll make the change.